app/views/notifications/_broadcast.html.erb

```diff
@@ -1,14 +1,12 @@
 <% json_data = notification.json_data %>
-<% if notification.action == "Onboarding" %>
-  <% user = User.find_by_id(json_data["user"]["id"]) %>
-  <% cache "broadcast-onboarding-notification" do %>
-    <a href="<%= json_data["user"]["path"] %>" class="small-pic-link-wrapper">
-      <div class="small-pic">
-        <img src="<%= ProfileImage.new(user).get %>" alt="link to <%= json_data["user"]["username"] %>'s profile">
-      </div>
-    </a>
-    <div class="content notification-content broadcast-content">
-      <%= json_data["broadcast"]["processed_html"].html_safe %>
+<% user = User.find_by(id: json_data["user"]["id"]) %>
+<% cache "broadcast-html-#{notification.json_data['broadcast']['title']}" do %>
```

Comment:
Hmm, I guess this new cache key wouldn't have issues with old broadcast notifications since those notifications won't have `json_data['broadcast']['title']` available and we only had one type of broadcast notification anyway.

Reply:
Yeah it should all be good. The old title (as passed in via `json_data` will be `nil`) which should interact fine here.